sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java

```diff
@@ -172,7 +172,9 @@ public static Windowing fromCalciteStuff(
             throw new CannotBuildQueryException(window, aggregateCall);
```

Comment:
I wonder why didn't this check have failed if the agg column was renamed to `:agg` ?